tests/default/query/datasources.yaml

```diff
@@ -0,0 +1,95 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test various operations of the OpenSearch Query Datasources API.
+version: '>=2.7'
+
+prologues:
+  - path: /_plugins/_query/_datasources/{datasource_name}
+    method: DELETE
+    parameters:
+      datasource_name: test_datasource
+    status: [204, 404]
+chapters:
+  - synopsis: Create a new query datasource.
+    path: /_plugins/_query/_datasources
+    method: POST
+    request:
+      payload:
+        name: test_datasource
+        description: ''
+        connector: PROMETHEUS
+        allowedRoles: []
+        properties:
+          prometheus.uri: 'http://localhost:9090'
```

Comment:
This (and a few other instances below) is the URL that gets tripped by the link checker. Change it to `https://...`

I am surprised the URL is actually not checked, is this expected? I'd think the API fails if prometheus is not available?

Reply:
It sound like it should fail, but I tested API in the playground with turned off prometheus and got 200 without errors